bundles/org.openhab.core.persistence/src/main/java/org/openhab/core/persistence/ModifiablePersistenceService.java

```diff
@@ -46,6 +46,7 @@ public interface ModifiablePersistenceService extends QueryablePersistenceServic
      */
     void store(Item item, ZonedDateTime date, State state);
 
+    // TODO: add a method for storing with alias to be in line with the ordinary store methods?
```

Comment:
It requires changes to all existing persistence services tat implement `ModifiablePersistenceService`. I have added it and will fix the persistence services once we are done here to prevent merge conflicts due to other changes.